src/com/rcv/Main.java

```diff
@@ -150,9 +150,8 @@ static String executeTabulation(ElectionConfig config) {
         } else {
           Logger.severe("No cast vote records found.");
         }
-      } catch (Exception exception) {
-        encounteredError = true;
-        Logger.severe("ERROR during tabulation: %s", exception.toString());
+      } else {
+        Logger.severe("Skipping tabulation due to source file errors.");
```

Comment:
Why move to this from the previous try / catch logic? The former seemed to be able to handle a broader array of situations and provide a more informative error message.

Reply:
The try/catch logic was just catching everything indiscriminately, which means at best we'd show a generic message and the raw exception -- which isn't a great user experience. Perhaps we'll need to add back a catch-all handler eventually, but I'd like to try to cover as many specific error conditions as possible in the code so that we can optimize the error messages that we show.

Reply:
Agreed, but at the moment if anything falls outside of the conditions you're checking for, the whole thing just dies, which is an even worse experience, right? I'd recommend adding back the sub-optimal UX of seeing the raw exception just in case something does happen to fall outside.

Reply:
Hmm, actually, I'm having trouble finding any way to make it die now. Let's see how it goes...